custom_components/tahoma/climate_deh.py

```diff
@@ -40,7 +36,7 @@ def supported_features(self) -> int:
     @property
     def temperature_unit(self) -> str:
         """Return the unit of measurement used by the platform."""
-        return UNIT_PERCENTAGE
+        return TEMP_CELSIUS
```

Comment:
The state is not in percent?

Reply:
it doesn't work... HA complains about it not being a valid temperature unit, and I didn't manage to to override every function that checks that...